README.md

```diff
@@ -63,6 +63,94 @@ You can configure the scheduler on the simulator through KubeSchedulerConfigurat
 
 [Scheduler Configuration | Kubernetes](https://kubernetes.io/docs/reference/scheduling/config/)
 
+You can pass a KubeSchedulerConfiguration file via the environment variable `KUBE_SCHEDULER_CONFIG_PATH` and the simulator will start kube-schedule  with that configuration.
```

Comment:
```suggestion
You can pass a path to KubeSchedulerConfiguration via the environment variable `KUBE_SCHEDULER_CONFIG_PATH` and the simulator will start kube-scheduler with that configuration.
```

Reply:
Oops... This line is probably an unrelated line engulfed by bad rebase. (But I fixed)